src/Game.java

```diff
@@ -48,12 +54,19 @@ public int takeATurn(Player whoseTurn){
 		boolean keepGoing = true;
 		printStartRoundMessage(whoseTurn);
 		while(keepGoing){
-			int roll = die.nextInt(7);
+			//int roll = die.nextInt(7);
+			//Cause: The nextInt method gives values between 0 and 6, therefore there is a chance to roll a 0.
+			int roll = die.nextInt(6) + 1;
 			String spin = spinner.spin();
 			System.out.println(roll+ " "+ spin);
 			
 			if(roll == LOSER_ROLL){
 				System.out.println("Lose a turn.");
+				//Cause: The above if statement accounted for rolling a 1, however it did not take into consideration checking for a grunt.
+				if(spin == LOSER_SPIN.toUpperCase()){
```

Comment:
This fix introduces copied code.  Better to re-arrange the code than to copy it.